tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify-all.yml.j2

```diff
@@ -3,6 +3,12 @@
     kind: Task
     name: mas-devops-ocp-verify-all
   params:
-    {{ lookup('template', 'taskdefs/common/cli-params.yml.j2') | indent(4) }}
```

Comment:
don't we still need taskdefs/common/cli-params.yml.j2' as it is still in the task?

Reply:
oh oh... yeah thanks for catching that, i'll have it fixed.